.github/workflows/simorgh-misc-checks.yml

```diff
@@ -27,6 +28,9 @@ jobs:
       - name: Install Node Modules
         run: npm ci
 
+      - name: Chromatic UI Tests # we should only run these one, they currently run on codebuild
```

Comment:
We can remove the comment from here, since we're not running CodeBuild for PRs anymore

Reply:
done